src/Circlereader/util/optionsAttribute.ts

```diff
@@ -0,0 +1,629 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
> Options other than 51 are made except for those because there is no attribute internally. :)

I see it :)
Then it would be better add them as comment, for example,
```typescript
// import {SpaceToBatchNDOptions} from '../circle-analysis/circle/space-to-batch-n-d-options';
```

If you want to apply this at another PR, please leave TODO comment and create another PR soon!

Reply:
> > Options other than 51 are made except for those because there is no attribute internally. :)
> 
> I see it :) Then it would be better add them as comment, for example,
> 
> ```ts
> // import {SpaceToBatchNDOptions} from '../circle-analysis/circle/space-to-batch-n-d-options';
> ```
> 
> If you want to apply this at another PR, please leave TODO comment and create another PR soon!

OK. I'll do it in another PR:)